lib/jekyll-compose/file_creator.rb

```diff
@@ -0,0 +1,33 @@
+class Jekyll::Compose::FileCreator
```

Comment:
Dunno, I guess I was just checking if I could. It seemed a little silly to indent the whole class two extra tab stops for no reason. It's also a little silly though to obscure the class name with constant scoping though.
I'll change it.